sdk/identity/identity/src/credentials/workloadIdentityCredentialOptions.ts

```diff
@@ -12,15 +12,15 @@ export interface WorkloadIdentityCredentialOptions
   /**
    * ID of the application's Azure Active Directory tenant. Also called its directory ID.
    */
-  tenantId: string;
+  tenantId?: string;
```

Comment:
Do we need the `WorkloadIdentityDefaultCredentialOptions` if now we're making all things optional in `WorkloadIdentityCredentialOptions`?

Reply:
We could, there was one place where TS was giving an error. Will show it you on the screenshare.